DuckDuckGo/AppDelegate.swift

```diff
@@ -66,6 +67,10 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
         #endif
 
+        CrashCollection.run { params in
+            Pixel.fire(pixel: .dbCrashDetected, withAdditionalParameters: params)
```

Comment:
Do you happen to know how long the OS will keep crashes around locally? I'm wondering how big of a spike we'll see once this goes live to customers, where there are presumably many of these reports cached and ready to deliver via MetricKit.

I don't think it's a blocker as it's a one-time issue that'll stabilize quickly, but would love to know what to expect.